### DISCUSSION TITLE
看了UP的linqpad视频推荐，本人也去下了一个试试，为啥我的输出框里面点击运行后啥都不显示？

### DISCUSSION THREAD

**Discussion by AniFacer** [BYJRK/DotNet-Discussions#127] [通用]
![image](https://github.com/BYJRK/DotNet-Discussions/assets/44011463/a1805478-63fe-4a84-8ce5-2299e0ca5f9c)

**Comment by BYJRK** [comment]
任何情况下都没输出吗？你试试 C# Statements，然后用 `.Dump()` 方法，看看有没有输出？

**Reply by AniFacer** [reply]
对，都试过，控制台就是不输出结果，导出excel里面竟然是有值得

**Reply by BYJRK** [reply]
Results里面确定是空的吗？有没有可能是字体和背景色相同之类
